Cargo.toml

```diff
@@ -10,6 +10,10 @@ repository = "https://github.com/influxdata/flux-lsp"
 [package.metadata.wasm-pack.profile.release]
 wasm-opt = ["-Oz", "--enable-mutable-globals"]
 
+[features]
+default = ["strict"]
+strict = []
+
```

Comment:
Same as above. I'm sure the change is fine, I'd just like to know why it's needed.

Reply:
Same thing. If you look at `Cargo.toml` in the `flux` crates, they also have this change. What it does it say "I have a feature, called 'strict', that will be defined in the package itself." It allows you to opt in or out of features. In this case, the feature equates to `deny(warnings, missing_docs)`, and is on by default (it had gotten turned off by accident in flux, but that's being fixed).